### DISCUSSION TITLE
Change default background color for Yaml and Json

### DISCUSSION THREAD

**Discussion by arnaudroques** [plantuml/plantuml#801] [General]
Right now, the [default color is yellow](http://www.plantuml.com/plantuml/uml/RP4nRyCW48Lt_mh4x4WniHsfKw9k7RnaYgeEhi655eC5vrPHvV_UuwZR9GleldVtUC0fGX_WFNfNV1UCyT6425q_AVx0-9FNZfS9s_wVNYO3EcCJpzPy9WfXKOJwjrIAOowa32f61CaSGOIW59p_C1SL4qwo2t4onQx1m2s9dVfYLny-DevPWI9qrmtD7g5zz99ujT-BQdV0ZQEyKePPUwUwPNfL8PtuJ7Cv3gfmcsbIUxnSkL57xJcVO4tJx2fng3d9QtdVmO-Z6Uvwj7LB7dNJ7dyzS7sXT8Cs5X_3fMIPnDalCJcwnTfYBKxAIVgV7m00).
What about [changing it to white by default](http://www.plantuml.com/plantuml/uml/RP6nRi8m48PtFuMLkQWa490egg1i7LYOKDKXZG_iulWY-rA4AjwzFfks2qkY--x-Bx-KLQ36qsT09vQ1BXQU1LZem97y5bA-DEtfw75mQeCM_KAUjI5u4bUnVBoTyrbnC3xGjkcWMCZY5RKh7XZRvf_kEaCwOnFMrdmn9Jz0GisHnsgU1wKyX101ooC8v05e_OUBgfILKtANE4lvgZLquUKMpdAF_iGyfnk2jDZ_eXuZj1jKYPVZSLME9l5LnFKrTUWrEjWEtGTu_k8Rvt8TpjEbJqcD7R0pKvVESxzArdKzAgl9b9rilM_0my6qTnspwImvflLiVdF4vtjgrsfZbGV7pJ89FGw1ZMwmLbp52fpYt_e3)?

Or [even this](http://www.plantuml.com/plantuml/uml/RP6nRi8m48RtF8MLkQWa4900LL2sgc9XGbM741_4nV55zgK8LRnxVHTe5vP4_kx-px_aPQ3IqrT0bom2NIoy962X0KVg9r7gjQnEHu-Tqskqw6VgN1k2UPoy6mStjhTnQvvSayNpJS7Hz61ye4tPG3fJwHlMBdrYRCj_kcqCrJqsOMNDDrFo7GWgZtpCf_r1QmyXHC3o209v05hzuJJFLEwqsWfdAMzLXYuytC1PxT2VxZnGII23tHsr6A5TenQU3OTvDXZ5Nnh7L-dGrkXWqpLxy7pZ1-UaZkJfqagonWR4ATJ9UjylLqLH3B9yD6QdM1yQy70mrKF7P3mHnxYOJ6-E-FsKTbLjhFRWk5bFGejTOAFhh8tlimId-Qb_0G00).

Does is improve readability ?

**Comment by swmatisa** [comment]
*>What about changing it to white by default
<http://www.plantuml.com/plantuml/uml/RP6nRi8m48PtFuMLkQWa490egg1i7LYOKDKXZG_iulWY-rA4AjwzFfks2qkY--x-Bx-KLQ36qsT09vQ1BXQU1LZem97y5bA-DEtfw75mQeCM_KAUjI5u4bUnVBoTyrbnC3xGjkcWMCZY5RKh7XZRvf_kEaCwOnFMrdmn9Jz0GisHnsgU1wKyX101ooC8v05e_OUBgfILKtANE4lvgZLquUKMpdAF_iGyfnk2jDZ_eXuZj1jKYPVZSLME9l5LnFKrTUWrEjWEtGTu_k8Rvt8TpjEbJqcD7R0pKvVESxzArdKzAgl9b9rilM_0my6qTnspwImvflLiVdF4vtjgrsfZbGV7pJ89FGw1ZMwmLbp52fpYt_e3>?
Does is improve readability ?*
For me YES

**Comment by The-Lum** [comment]
Hello all,

You can change, but do not avoid to change also the `highlight` accordingly.

See some tests here:

1. Current behaviour
```puml
@startjson
#highlight "color" / "0"
{
   "fruit":"Apple",
   "size": "Large",
   "color": ["Red", "Green"]
}
@endjson
```
>[![](http://www.plantuml.com/plantuml/svg/SoWkIImgoIhEp-DIpiXCpyW1uX85fUJydFmY9GLz1IK39QvgBWK51QMqejBC4YKh9SU2WfnK9HsmM75cLQgIbOAIJs9HEaGCgXKe6AqKb9gYfAEWv5wKcfgd5CjLo-MGcfS2iWe0)](http://www.plantuml.com/plantuml/uml/SoWkIImgoIhEp-DIpiXCpyW1uX85fUJydFmY9GLz1IK39QvgBWK51QMqejBC4YKh9SU2WfnK9HsmM75cLQgIbOAIJs9HEaGCgXKe6AqKb9gYfAEWv5wKcfgd5CjLo-MGcfS2iWe0)

2. Change to white background
```puml
@startjson
<style>
element {
  BackgroundColor: white;
}
highlight {
  BackgroundColor: silver;
}
</style>
#highlight "color" / "0"
{
   "fruit":"Apple",
   "size": "Large",
   "color": ["Red", "Green"]
}
@endjson
```
>[![](http://www.plantuml.com/plantuml/svg/TOyn2y8m48Nt_8hZkHRg78jK7LoSNCMXj6SRZKcvf8gA_zr62auEjpo-znsl3B6IU0xUgIB4X-MbOijNTf5UYcXTrPTM_E2QZRTUDDqx4tcXtgepRMV7-yC5Osyi2IpooJlxLL0d2fGJvb19G3Z9O28qLdrl6Tat2-R9q8HT9UsKJLLD1-ovGKRO2hF3SNnLiclIb0y0)](http://www.plantuml.com/plantuml/uml/TOyn2y8m48Nt_8hZkHRg78jK7LoSNCMXj6SRZKcvf8gA_zr62auEjpo-znsl3B6IU0xUgIB4X-MbOijNTf5UYcXTrPTM_E2QZRTUDDqx4tcXtgepRMV7-yC5Osyi2IpooJlxLL0d2fGJvb19G3Z9O28qLdrl6Tat2-R9q8HT9UsKJLLD1-ovGKRO2hF3SNnLiclIb0y0)

3. Or plain...
```puml
@startjson
<style>
element {
  BackgroundColor: white;
  LineColor: black;
}
highlight {
  BackgroundColor: silver;
}
</style>
#highlight "color" / "0"
{
   "fruit":"Apple",
   "size": "Large",
   "color": ["Red", "Green"]
}
@endjson
```
>[![](http://www.plantuml.com/plantuml/svg/TO-npe8m54RtFiNDzwya_7CX1dLmON8r3WXNgDQMj4MZndUNcYPE3brEpkcNM_dGk73oreZIXxlcXM3D5pQ1de9erRJdtjd9T6khhPDq6rJWGX3Lod1Y1prRXNY9GVM3dj-FrYjzPHV5CazRVzy4RRH0EU4V8dv0EBf91KWinr4piW_pwi6GXBfnVM8fbRJ3bZjaX8rZDjZFKnMRBfxt1W00)](http://www.plantuml.com/plantuml/uml/TO-npe8m54RtFiNDzwya_7CX1dLmON8r3WXNgDQMj4MZndUNcYPE3brEpkcNM_dGk73oreZIXxlcXM3D5pQ1de9erRJdtjd9T6khhPDq6rJWGX3Lod1Y1prRXNY9GVM3dj-FrYjzPHV5CazRVzy4RRH0EU4V8dv0EBf91KWinr4piW_pwi6GXBfnVM8fbRJ3bZjaX8rZDjZFKnMRBfxt1W00)

Another question will be:
- Which colour for the document background?
_White or transparent?_

Regards.
